src/utils/fetchClient.ts

```diff
@@ -1,34 +1,30 @@
 const BASE_URL = 'https://mate.academy/students-api';
 
-// a promise resolved after a given delay
 function wait(delay: number) {
-  return new Promise(resolve => {
+  return new Promise((resolve) => {
     setTimeout(resolve, delay);
   });
 }
 
-// To have autocompletion and avoid mistypes
 type RequestMethod = 'GET' | 'POST' | 'PATCH' | 'DELETE';
 
 function request<T>(
   url: string,
   method: RequestMethod = 'GET',
-  data: any = null, // we can send any data to the server
+  data: any = null,
```

Comment:
![image](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/56752562/8c6e3591-06a1-4a55-9a1a-d30ada7005c6)
Fix linter error